api/app/Notify/Client.php

```diff
@@ -3,10 +3,12 @@
 namespace App\Notify;
 
 use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
```

Comment:
Not referenced
```suggestion
```

Reply:
Removed in [removed unused import](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5350/commits/11de08a3cbc11f6efd872feaaf5582bb4a9a62d5)